package.json

```diff
@@ -1,7 +1,17 @@
 {
-  "name": "paper-autocomplete",
   "description": "Material Design autocomplete component.",
-  "version": "3.7.0",
+  "keywords": [
+    "web-component",
+    "polymer",
+    "paper-autocomplete"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ellipticaljs/paper-autocomplete.git"
+  },
+  "homepage": "https://github.com/ellipticaljs/paper-autocomplete/",
+  "name": "@cbar/paper-autocomplete",
```

Comment:
are you planning to fork this component?

Reply:
Yes! This was actually pushed to my fork - forgot the branch was tied to a PR. Reverting now, cheers.